purchase_requisition_order_remaining_qty/models/purchase_requisition_line.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class PurchaseRequisitionLine(models.Model):
+    _inherit = "purchase.requisition.line"
+
+    propose_qty = fields.Float(
+        compute="_compute_propose_qty", store=True, string="Propose Qty"
+    )
+
+    @api.depends(
+        "requisition_id.type_id",
+        "requisition_id.purchase_ids",
+        "requisition_id.purchase_ids.order_line",
+        "requisition_id.purchase_ids.order_line.product_id",
+        "requisition_id.purchase_ids.order_line.product_qty",
+        "requisition_id.purchase_ids.state",
+    )
+    def _compute_propose_qty(self):
+        for item in self:
+            item.propose_qty = sum(
+                item.requisition_id.filtered(
+                    lambda x: x.type_id.quantity_copy == "remaining_qty"
```

Comment:
Why restrict to this type? The proposed quantity is going to be the same on any kind of selection, so let it computed for all.